.github/CONTRIBUTOR_GUIDE/CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+
+1. **Visual Studio Code** (recommended for script development)
+2. **Recommended VS Code Extensions:**
+   - [Shell Syntax](https://marketplace.visualstudio.com/items?itemName=bmalehorn.shell-syntax)
+   - [ShellCheck](https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck)
+   - [Shell Format](https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format)
+
+### Important Notes
+- Use [AppName.sh](https://github.com/community-scripts/ProxmoxVE/blob/contributor_guide/.github/CONTRIBUTOR_GUIDE/ct/AppName.sh) and [AppName-install.sh](https://github.com/community-scripts/ProxmoxVE/blob/contributor_guide/.github/CONTRIBUTOR_GUIDE/install/AppName-install.sh) as templates when creating new scripts.
+- The call to `community-scripts/ProxmoxVE` should be adjusted to reflect the correct fork URL.
+
+---
+
+# 🚀 The Application Script (ct/AppName.sh)
+
+- You can find all coding standards, as well as the structure for this files [here](https://github.com/community-scripts/ProxmoxVE/blob/contributor_guide/.github/CONTRIBUTOR_GUIDE/ct/AppName.md).
+- These Scripts are responsible for container creation, setting the necessary variables and handles the update of the application.
+
+---
+
+# 🛠 The Installation Script (install/AppName-install.sh)
+
+- You can find all coding standards, as well as the structure for this files [here](https://github.com/community-scripts/ProxmoxVE/blob/contributor_guide/.github/CONTRIBUTOR_GUIDE/install/AppName-install.md).
+- These scripts are responsible for the installation of the application.
+
+---
+
+## 🚀 Building Your Own Scripts
+
+Start with the [template script](https://github.com/community-scripts/ProxmoxVE/blob/contributor_guide/.github/CONTRIBUTOR_GUIDE/install/AppName-install.sh)
+
+---
+
+## 🤝 Contribution Process
+
+### 1. Fork the Repository
+Fork to your GitHub account
+
+### 2. Clone Your Fork on your Pc 
+```bash
+git clone https://github.com/yourUserName/ForkName
+```
+
+### 3. Create a New Branch
+```bash
+git switch -c your-feature-branch
+```
+
+### 4. Change Paths in build.func install.func and AppName.sh
+You need to switch `https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main` to `https://raw.githubusercontent.com/[USER]/[REPOSITORY]/refs/head/[BRANCH]` to test your script in your repository. Before opening a Pull Request change all this back to point to the community-scripts Repository.
+
+### 4. Commit Changes (without build.func and install.func!)
+```bash
+git commit -m "Your commit message"
+```
+
+### 5. Push to Your Fork
+```bash
+git push origin your-feature-branch
+```
+
+### 6. Create a Pull Request
+Open a PR from your feature branch to the main repository branch. You may only include your **AppName.sh**, **AppName-install.sh** and **AppName.json** files in the pull request.
+
+---
+
+## 📚 Pages
+
+- [Function-Overview](https://github.com/community-scripts/ProxmoxVE/wiki/Function_Overview)
```

Comment:
This page points to nowhere.

Reply:
This was put there by @MickLesk. Is there something planed (like documentation from build.func/install.func or something like that?) from your side ore should we remove it for now?

Reply:
This raises a general question: Is the wiki in use or should it be used? The wiki itself is also a git repo. However, the pull request flow (incl. reviews and quality gate) is not lived there.

Having docs in markdown + wiki can lead to confusion, as people asks themselves "which one is now up to date?".

It may be helpful to commit to the one or the other.